widget: Improve the Connect Now button

staticfox opened the pull request:
Make the inactive state color match the active state
Remove the blue glow surrounding the border and text

---
# Before:
### Active, enabled

![](http://i.imgur.com/WWwrkNj.png)
### Inactive, enabled

![](http://i.imgur.com/adyiHkg.png)
## Active, disabled

![](http://i.imgur.com/wL00gwn.png)
# After:
### Active, enabled

![](http://i.imgur.com/CrVlyUW.png)
### Inactive, enabled

![](http://i.imgur.com/cSY5jeg.png)
### Active, disabled

![](http://i.imgur.com/BqhVDAF.png)

staticfox committed:
widget: Improve the Connect Now button

Make the inactive state color match the active state
Remove the blue glow surrounding the border and text

un1versal commented:
I dont like the inactive enabled states, they are too dissimilar and dont make much sense to me, why we need this 3rd state?

staticfox commented:
Because it existed before but was only made evident for when you changed the button to green and the inactive state remained blue (unless you intended it to be this way?)

edit: The 3rd state has always exited whether you realized it or not, I updated the button color to match the active state and changed the text to black because if the user is clicked away, you generally want to keep the button more subtle so they can focus on their active window. (like in photography, your eye goes the whitest point so if a user is scoped in to another window that isn't kvirc, it would make sense to darken the text)

un1versal commented:
what conditions is this 3rd one you mention visible?

Select the network
![capture](https://cloud.githubusercontent.com/assets/3521959/10755137/c5d59b7c-7c93-11e5-9984-eb6fe64facb0.PNG)

Select server 
![capture1](https://cloud.githubusercontent.com/assets/3521959/10755136/c5d22c12-7c93-11e5-82db-8d5f066154e8.PNG)

I cant see any 3rd thing anywhere, what am I missing?

staticfox commented:
select a server then click on another program like your web browser with the server browser still in view

wodim commented:
Commit these things yourself, it is not like anybody will complain

un1versal commented:
I was in the middle of testing this.

un1versal commented:
OK now I had a chance to review and test some adjustments made in https://github.com/kvirc/KVIrc/commit/decd95c446e2df9e269781c7ea0ee738c2f9007b as follows

active - focus
![active - focus](https://cloud.githubusercontent.com/assets/3521959/10759178/c71f2bfa-7cad-11e5-9e4b-dd1786afb28e.PNG)

active - unfocus
![active - unfocus](https://cloud.githubusercontent.com/assets/3521959/10759177/c71bebc0-7cad-11e5-9b38-9f683ded3cf9.PNG)

inactive
![inactive](https://cloud.githubusercontent.com/assets/3521959/10759176/c718cf94-7cad-11e5-9bd8-bdf78ed56ac8.PNG)

@blindsighttf2 Do you agree?
